db/migrations/01_create_peeps_table

```diff
@@ -0,0 +1,31 @@
+-- AUTOMATIC timestamps with PostgreSQL
```

Comment:
I have different way to do the timestamp, as I did not know I could do it in the database side.